### ISSUE TITLE
[🐞]: Abnormals wood types not generating for various blocks

### ISSUE THREAD

**Issue by yakisobadayo** [MehVahdJukaar/WoodGood#546]
### Before Continuing:

- [X] Always check you are using the latest version of the mods and its dependencies
- [X] Remove mod that enhances Minecraft: Optifine, Sodium, others. The issue still persists.
- [X] If you are unsure which mod is the culprit.
Disable all of your mods and enable them 1-2 mods each time to isolate the culprit
- [X] Confirm that there is no existing issue with a similar description submitted in the list of issues.

### Version

1.20.1

### Loader

Forge

### EveryCompat Version

everycomp-1.20-2.6.65

### Moonlight Lib Version

moonlight-1.20-2.12.6-forge

### Issue with mods

Autumnity - maple
Upgrade Aquatic - river, driftwood
(Both of these require Blueprint)

Quark - bookshelf, chest, ladder
Architect’s Palette - boards

### Issue Detail

Basically, stuff like Quark’s bookshelves and chests don’t generate, I suspect because wood variants for these blocks already exist in the mods but are hidden if Abnormal’s Woodworks isn’t installed.

Architect’s Palette boards don’t generate either; however slab and stair variants do. The slabs and stairs have crafting recipes, but they use the Woodworks boards, which are unobtainable without Woodworks. It would be nice to have these blocks available without having to install Woodworks.

**Comment by Xelbayria** [comment]
Alright. 
I know what's the problem with `Architect’s Palette boards`. i just fixed this in 1.19.2. and the other blocks, i'll take a look

**Comment by Xelbayria** [comment]
>Autumnity - maple
Upgrade Aquatic - river, driftwood
(Both of these require Blueprint)
>
>Quark - bookshelf, chest, ladder
Architect’s Palette - boards

ok i will need a clear description: 

Furniture + woodMod
- Quark + Upgrade Aquatic = no bookshelf, chest, ladder
- Architect's Palette + ??? wood mod = no boards

The problem with boards (it's from Woodworks) and the woodMod is Architect's Palette. that's what I fixed in 1.19.2

**Comment by yakisobadayo** [comment]
Wood mods = Upgrade Aquatic and Autumnity
Furniture mods = Quark and Architect’s Palette

Quark + Upgrade Aquatic and Autumnity = no bookshelf, chest, ladder
Architect’s Palette + Upgrade Aquatic and Autumnity = no boards

When these mods are installed, maple, river, and driftwood variants of the quark blocks and architect’s palette boards don’t appear. Hope this is enough information, if not I’ll try to update when I get home

**Comment by Xelbayria** [comment]
> Quark + Upgrade Aquatic and Autumnity = no bookshelf, chest, ladder
Architect’s Palette + Upgrade Aquatic and Autumnity = no boards

got it, i know what to look at.

**Comment by Xelbayria** [comment]
- [x] **Architect's Palette (AP)** + **Upgrade Aquatic (UA)**
	- [x] river_boards
	- [x] driftwood_boards

- [x] **AP** + **Autumnity (AUT)**
	- [x] maple_boards

---

FYI, both of **AUT** & **UA** already have chests, bookshelf and ladder. but they don't have recipes. the best way to solve this is to report it to the dev and use a temporary solution with CraftTweaker or KubeJS. 

**Quark** & **Woodworks** are very similar where they added variant wood for chest, bookshelf, ladder, so on.   **Woodworks** are made by _Abnormal Team_. **UA** & **AUT** are made by the same team. 

for now, i'll see if i can fix the boards issue on **AP**'s side. 

### UPDATED: 

The fix for **UA** and **AP** are already in the code. but I fixed the missing boards from **AUT** with **AP**. It'll be in **v2.6.66**

so about the bookshelf, chest, and ladder. They are not really required because they are not used as ingredient unlike **AP**'s board is. You can use the chest, bookshelf, and ladder in **UA** & **AUT**. The missing recipes will be fixed tho so you can craft them. 

let me know your opinion on this.

**Comment by yakisobadayo** [comment]
Yes, that sounds good to me. All I wanted is to be able to craft the bookshelf variants and such without having to install Woodworks. Thank you!

**Comment by Xelbayria** [comment]
oh i need to clarify my mistake

> The missing recipes will be fixed tho

it should be _"the missing recipes need to be fixed, tho"_, so that mean it needed to be fixed on your side using CraftTweaker or KubeJS. you should let the DEV know about the missing recipe. The only fix is for the missing board from **Autumnity**. it will be in v2.6.66. 

Thanks for the replying

**Comment by yakisobadayo** [comment]
Ohh I understand, no worries. I will try and let the devs know, however I believe the hidden recipes are intentional, as they are enabled when Woodworks is present. As Quark also adds the same vanilla chest and bookshelf variants Woodworks adds, I avoided installing Woodworks as well as I didn’t want to have duplicates. But I will try to add the missing recipes with KubeJS for now

**Comment by Xelbayria** [comment]
If you want to add Woodworks to fix the recipes. I can fix this.  I can make it that EveryComp not creating the duplicated block. if Woodworks are installed with **UA** and **AUT**. this will take some works to fix. Let me know.

**Comment by yakisobadayo** [comment]
Ah, no need, I found out that Quark already disables its wood variants when Woodworks is installed. Thank you though!

**Comment by Xelbayria** [comment]
oh yes, Quark's config. I forgot it has the ability to hide variants. Good to know. Can you confirm that the issue is fixed with the configs?  I may not need to apply the fix to this.

**Comment by yakisobadayo** [comment]
Everything looks to be working perfectly except that there are no regular chests for the Quark wood types, but I see that there's an issue opened for that already

**Comment by Xelbayria** [comment]
If you are talking about `No Woodworks chests generated for Quark wood types #549`, then yea it's a bit tricky issue. I wondered if **Quark**'s config is the reason for why the recipe on **Quark**'s side for chest and trapped_chest not working. There is no confirmation. 

I tested it without **EveryComp** and a clean config for **Quark**, the recipe is not working. it's an issue on **Quark**'s side. If I am wrong, then I would like to know what is the real cause. It would be good to know, tho
